README.md

```diff
@@ -62,30 +69,43 @@ bundle install --path vendor/bundle
 npm i babel-minify
 ```
 
-## Running locally :running:
+## Building :running:
 
-Ubuntu/WSL/MacOS:
+Docker (Windows/Linux/MacOS):
+
+```BASH
+docker run -p 4000:4000 -v $(pwd):/twofactorauth
+```
+
+Snap/Manual:
 
 ```bash
-# Generating regional sites (Optional)
-ruby ./_deployment/regions.rb
+# Create _data/all.json
+ruby ./scripts/join-entries.rb > _data/all.json
```

Comment:
To be on the safe side, `_data/all.json` should also be included in `.gitignore` then.